app/src/main/res/values/strings.xml

```diff
@@ -98,4 +98,6 @@
     <string name="msg_no_permission">No permission</string>
     <string name="msg_edit_success">You have successfully edit this booking</string>
     <string name="msg_cant_find_date">Can\'t find booking day</string>
+    <string name="title_information">Information</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_information` appears to be unused
 Unused resources make applications larger and slow down builds.